docs/ko/faq/why-kotlin-jdsl-returns-nullable-list-in-the-spring-support-module.md

```diff
@@ -0,0 +1,40 @@
+# 왜 Kotlin JDSL은 Nullable한 반환 타입을 허용하나요?
+
+Kotlin JDSL을 사용하면 Type Safe하게 쿼리를 작성할 수 있게 도와주지만
+다건의 데이터를 반환할 때 non-null한 데이터 또한 null을 허용하는지 궁금증을 가질 수 있습니다. (예를 들어 `findSlice`의 `Slice<T?>`, `findPage`의 `Page<T?>`)
+이 글은 Kotlin JDSL에서 nullable한 반환 타입을 허용하는 이유를 설명합니다.
+
+## Nullable 반환 타입의 필요성
+
+Kotlin JDSL이 nullable한 반환 타입을 지원하는 이유는 JPQL 쿼리의 작성 방법에 따라 반환되는 리스트 항목 중 null 값이 포함될 수 있기 때문입니다.
+가장 간단한 예로는 DTO Projection을 사용하지 않고, column이나 entity를 조회할 때를 들 수 있습니다.
+
+아래 표는 Kotlin JDSL을 사용할 때 nullable한 반환 타입이 발생할 수 있는 몇 가지 경우를 나타냅니다.
+
+| 항목	            | null 여부 | 	이유                                                                                                   |
+|----------------|---------|-------------------------------------------------------------------------------------------------------|
+| DTO Projection | 	X      | 	모든 ROW에 대한 생성자 호출하기 때문에 객체가 생성되어 null을 허용하는 결과가 나오지 않음<br/>DTO의 필드가 null일 수는 있으나 DTO 객체가 null일 수는 없음 |
```

Comment:
```suggestion
| DTO Projection | 	X      | 	모든 ROW에 대한 생성자를 호출하기 때문에 객체가 생성되어 null을 허용하는 결과가 나오지 않음<br/>DTO의 필드가 null일 수는 있으나 DTO 객체가 null일 수는 없음 |
```
i think this is correct.

Reply:
Thank you for the detailed review! I will edit it. ☺️